umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -1,142 +1,247 @@
 
+The Management Companion is a microservice that runs in conjunction with the UMH.
+It plays a crucial role in linking the instance to the Management Console. This
+includes gathering the status of the device and the various components within
+the system.
 
-1. Download and install [Docker Desktop](https://www.docker.com/products/docker-desktop)
-   for your operating system. This is only 
-   necessary for local installations.
-   
+The diagram below illustrates the communication flow between the Management
+Console and the instance:
 
-2. Follow the tutorial in the Management Console to create your first local 
-   instance of the United Manufacturing Hub. Once the installation is finished,
-   press the blue **Finish** button the right bottom corner of the Console. 
+![Communication between the Management Console and the instance](/images/getstarted/managingTheSystem/simpleInfrastructureDiagram.png?width=80%)
 
+## Overview of Your Instances
 
-{{% notice info %}}
-A local installation functions similar to an installation on an external device,
-like an edge device or a VM. The benefit is, that you can easily set it up
-without any additional hardware and learn how to use it with simulated data.
+On the left side of the Management Console, you can view the list of your
+instances. If you have just installed the UMH, you should see only one instance
+in the list.
+
+The status of your instance is indicated by color: green means everything is
+working properly, while yellow indicates that there may be an issue.
+
+The **Messages Received** statistic shows the number of messages received by you
+from the instance since you opened the Management Console. It is usually a good
+indicator of the health of the Companion and the latency of the connection between
+you and the instance.
+
+## Monitoring the Instance's Status
+
+From the instance dashboard, in the overview tab, you can view the status of
+your instance. There are multiple interfaces that display the status of each
+component of the system.
+
+### Modules
+
+A _Module_ refers to a group of workloads in the United Manufacturing Hub
+responsible for specific tasks. For example, the _Data Source_ module collects
+data from various sources and is composed of multiple microservices.
+
+In the **Modules** tab, you can view the status of each module. If a module is
+not healthy, it means that one or more of its microservices are not functioning
+properly.
+
+### System
+
+In the **System** tab, you can view the resource usage of your device, as well
+as some system's information.
+
+If there is an overload on the device, you can view it here. An overloaded device
+is unable to handle the workload, and you should consider upgrading the device.
+
+### Data
+
+In the **Data** tab, you can see an overview of the data infrastructure, including
+the number of messages going through the message broker, those stored in the
+database, and messages received by each data source.
+
+### Connection
```

Comment:
This will be in the future called Asset Management. Maybe we can already explain its role here:

https://github.com/orgs/united-manufacturing-hub/projects/8/views/8?pane=issue&itemId=44473179